### PR TITLE
Create lxqt-hibernate_it.desktop

### PR DESCRIPTION
**Pull request by stefonarch**

### CODE REVIEW

File: lxqtleave/translations/lxqt-hibernate_it.desktop [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+GenericName=Hibernate
+Comment=hibernate the machine
+Exec=lxqt-leave --hibernate
+Icon=system-suspend-hibernate
+Categories=LXQt;X-Leave
+OnlyShowIn=LXQt
+
+#TRANSLATIONS_DIR=../translations
+
+# Translations
```
Comment (palinek) [inline lxqtleave/translations/lxqt-hibernate_it.desktop]: Please, remove all universal keys and leave here only the translated ones (\[it\]).

Comment (stefonarch) [inline lxqtleave/translations/lxqt-hibernate_it.desktop]: There has to be an issue with this, looking at the italian desktop files
with only the #translation part they are not built correctly
http://i.imgur.com/jgCxW4I.png
The universal keys are all missing:

$ cat lxqt-shutdown_it.desktop
#Translations
Name[it]=Arresta
Comment[it]=Spegnere il computer

The  desktop files  in other languages are in source all complete
including the universal part.


Il 07/04/2015 00:15, palinek ha scritto:

Comment (palinek) [inline lxqtleave/translations/lxqt-hibernate_it.desktop]: The final \<app\>.desktop file is generated as concatenation of \<app\>.desktop.in file and translations/\<app\>\_\*.desktop file(s) (only \*\[\*\] keys are filtered). So there is no need for `translations` files to list all the universal keys.

**Comment by paulolieuthier** [comment]
Merge in 99911b0fea752c2a64068ee52f9bad54c780efe4.

**Pull request closed without merging**